pkg/apis/peakq/v0alpha1/types.go

```diff
@@ -130,6 +130,9 @@ type QueryTemplateList struct {
 type RenderedQuery struct {
```

Comment:
This might need the `Values` map as well in order to be able to reconstruct the Render operation from a RenderedQuery that has been saves somewhere (Since that isn't in the replacements).